crates/gateway-types/src/reply.rs

```diff
@@ -336,6 +354,8 @@ pub mod transaction {
                 l2_to_l1_messages: Faker.fake_with_rng(rng),
                 transaction_hash: Faker.fake_with_rng(rng),
                 transaction_index: Faker.fake_with_rng(rng),
+                execution_status: Faker.fake_with_rng(rng),
+                revert_error: Faker.fake_with_rng(rng),
```

Comment:
This can produce incorrect data because these fields aren't actually independent..